shared/src/test/scala/squants/thermal/TemperatureSpec.scala

```diff
@@ -47,32 +52,39 @@ class TemperatureSpec extends FlatSpec with Matchers {
     x.toKelvinScale should be(0)
     x.toFahrenheitScale should be(-459.67)
     x.toCelsiusScale should be(-273.15)
+    x.toScale(Rankine) should be(0)
```

Comment:
Why is this `toScale` unlike the other methods like `toKelvinScale`, etc. Should it be called `toRankineScale` instead?